lighthouse-core/lib/i18n/i18n.js

```diff
@@ -259,6 +259,20 @@ function getFormatted(icuMessageIdOrRawString, locale) {
   return icuMessageIdOrRawString;
 }
 
+/**
+ * @param {LH.Locale} locale
+ * @param {string} icuMessageId
+ * @param {*} [values]
+ * @return {string}
+ */
+function formatMessageFromIdWithValues(locale, icuMessageId, values) {
+  const icuMessageIdRegex = /(.* \| .*)$/;
+  if (!icuMessageIdRegex.test(icuMessageId)) throw new Error('This is not an ICU message ID');
+
+  const {formattedString} = _formatIcuMessage(locale, icuMessageId, '', values);
```

Comment:
empty fallback message concerns me a bit, maybe we should have `_formatIcuMessage` throw if we can't find the icuMessageId and the fallback message is undefined? Maybe we should use a placeholder string that's non-empty?

not sure what to do there 🤷‍♂